azure-pipelines.yml

```diff
@@ -18,27 +18,38 @@ parameters:
         command: smoke
         serenity: false
         dependsOn: UnitTest
+      - name: AcceptanceTest
+        command: acceptance
+        serenity: false
+        dependsOn: SmokeTest
       - name: FunctionalTest
         command: functional
         serenity: true
-        dependsOn: SmokeTest
+        dependsOn: AcceptanceTest
       tests_security:
       - name: ApiZapScan
         containerRegistry: ssprivatesbox.azurecr.io
         repository: hmi/apim-tests-owasp
 
-    # - name: dev
-    #   subscription: DTS-SHAREDSERVICES-DEV
-    #   group: HMI-APIM-BUILD-DEV
-    #   tests_gradle:
-    #   - name: UnitTest 
-    #     command: clean test
-    #   - name: SmokeTest
-    #     command: clean smoke
-    #     dependsOn: UnitTest
-    #   - name: FunctionalTest
-    #     command: clean functional
-    #     dependsOn: SmokeTest
+    - name: dev
+      subscription: DTS-SHAREDSERVICES-DEV
+      group: HMI-APIM-BUILD-DEV
+      tests_gradle:
+      - name: UnitTest
+        command: test
+        serenity: true
+      - name: SmokeTest
+        command: smoke
+        serenity: false
+        dependsOn: UnitTest
+      - name: AcceptanceTest
+        command: acceptance
+        serenity: false
+        dependsOn: SmokeTest
+      - name: FunctionalTest
```

Comment:
I dont think we will be executing Functional Tests in Dev...So this can be removed.....